README.md

```diff
@@ -41,7 +41,7 @@ We collect those metrics over time to get indicators of how the performance chan
 for a more detailed instruction how to read the results you should consider reading the k6 manual,
 specially the [end of test](https://k6.io/docs/results-output/end-of-test/) section.
 
-## Available test suits
+## Test Suits
```

Comment:
I didn't mean to remove the `Available`, but the `e` was missing in `Suits` - sorry for the confusion.

```suggestion
## Available Test Suites
```